.github/workflows/build.yml

```diff
@@ -171,6 +171,15 @@ jobs:
           MAKE: make -j2 --output-sync=recurse
           SAGE_NUM_THREADS: 2
 
+      - name: Imports
+        run: |
```

Comment:
```suggestion
        run: |
        if: always() && steps.build.outcome == 'success'
```

Reply:
I don't see the need to run this step if one of the other steps fails.

Reply:
Right now it's not being run because of a pyright error.

Reply:
... so you may want to explain why you think that this particular step should not be run, although all the following ones are being run...

Reply:
As I said before, in my opinion this step is not important enough nor does it add helpful information in case another step fails.